src/_rules/typography.js

```diff
@@ -10,4 +10,5 @@ export const typography = [
     ({ 'line-height': `var(--w-line-height-${size})` }),
   ],
   [/^leading-\[(.+)(rem|px)?\]/, ([, value, unit], context) => ({ 'line-height': resolveArbitraryValues(value, unit, context) })],
+  ['tabular-nums', { 'font-family': 'ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif', 'font-variant-numeric': 'tabular-nums' }],
```

Comment:
Missing class and requested from Dave.
It was in fabric here: https://github.com/fabric-ds/css/blob/01fd80a27461cf0d593911e974853f84963d0681/src/utilities.css#L13

Reply:
I found a TODO in the css-docs related page. https://warp-ds.github.io/css-docs/font-variant-numeric#applying-numeric-variants 👇 
`!TODO: Discuss the support for this in warp. We might to have to theme this somehow, these fonts might look too different to some of the brand fonts and perhaps need different weights to match them better to the brand font.`
Is it something we need to consider?

Reply:
Very good catch 🙏 I added a ticket: https://nmp-jira.atlassian.net/browse/WARP-171